pkg/virt-handler/vm_test.go

```diff
@@ -3301,6 +3304,93 @@ var _ = Describe("VirtualMachineInstance", func() {
 		})
 	})
 
+	Context("setMigrationProgressStatus", func() {
+		newDomainMigrationKubevirtMetadata := func(miguid types.UID, end *metav1.Time, completed, failed bool, mode v1.MigrationMode) *api.Domain {
+			d := api.NewMinimalDomainWithUUID("test", types.UID(1234))
```

Comment:
I am not 100% sure but looks like s390x lane does not like `types.UID(1234)` (With integer instead of string).
```suggestion
			d := api.NewMinimalDomainWithUUID("test", "1234")
```
The same applies to the other occurrences of `types.UID(1234)`